schematic/schemas/create_json_schema.py

```diff
@@ -15,13 +15,17 @@
 
 from schematic.schemas.data_model_graph import DataModelGraphExplorer
 from schematic.utils.schema_utils import get_json_schema_log_file_path
-from schematic.utils.validate_utils import rule_in_rule_list
 from schematic.utils.io_utils import export_json
-from schematic.schemas.constants import (
-    ValidationRule,
+from schematic.schemas.validation_rule_functions import (
+    ValidationRuleName,
     JSONSchemaType,
-    RegexModule,
-    TYPE_RULES,
+    filter_unused_rules,
+    check_for_rule_conflicts,
+    check_for_rule_duplicates,
+    get_in_range_parameters_from_rule,
+    get_regex_parameters_from_rule,
+    get_js_type_from_rule_list,
+    get_rule_from_rule_list,
 )
```

Comment:
The code for getting JSON Schema keywords from validation rules was moved to validation_rule_functions.py